specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2019-05-05-preview/AlertsManagement.json

```diff
@@ -922,10 +922,11 @@
   "parameters": {
     "subscriptionId": {
       "name": "subscriptionId",
-      "description": "Subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call.",
-      "type": "string",
       "in": "path",
-      "required": true
+      "required": true,
+      "type": "string",
+      "description": "The ID of the target subscription.",
+      "minLength": 1
```

Comment:
Why do we require 'minLength' property?

Reply:
Just following syntax from common file here:
https://github.com/Azure/azure-rest-api-specs/blob/f5f7b4691687afdfa1130526b1c34fa7720bc7e5/specification/common-types/resource-management/v1/types.json#L274-L281

It just indicates that it shouldn't be empty string.

Reply:
Makes sense.